vertx-web-openapi/src/main/java/io/vertx/ext/web/openapi/impl/OpenAPI3RouterBuilderImpl.java

```diff
@@ -307,6 +307,7 @@ public Router createRouter() {
         .solve() // If this optional is empty, this route doesn't need regex
         .map(solvedRegex -> router.routeWithRegex(operation.getHttpMethod(), solvedRegex.toString()))
         .orElseGet(() -> router.route(operation.getHttpMethod(), operation.getOpenAPIPath()))
+        .order(pathResolver.parameters.size())
```

Comment:
I wonder, is it better to preorder the `operations` field more than using `order`? I'm worried that the order, set as is here, might cause issues to other handlers ordering